addons/project/static/src/components/subtask_kanban_list/subtask_kanban_create/subtask_kanban_create.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<templates>
+    <div t-name="project.SubtaskCreate" class="subtask_create_input d-flex" owl="1">
+        <input type="text" title="Rename" 
+            class='border-0 border-bottom py-1'
+            t-ref='subtaskCreateInput'
+            t-att-value="state.name"
+            t-att-placeholder="placeholder"
+            t-on-input="_onInput"
+            t-on-click="_onClick"
+            t-on-blur="_onBlur"
+            t-on-change="_onNameChanged"
+            t-att-disabled="props.isReadonly"/>
+        <button t-on-click="_onSaveClick" class="ms-auto fw-bold btn btn-link py-0 px-2">
+            SAVE
```

Comment:
```suggestion
            Save
```
? 🧐

Reply:
Based on the save todo action in the systray
https://imgur.com/tQczD76

Reply:
Ah okay. :o